.github/labeler.yml

```diff
@@ -61,6 +61,7 @@
 
 "Melee":
   - "**/**melee**"
+    - data/json/techniques.json
```

Comment:
```suggestion
  - "**/melee**"
  - "**/**techniques**"
```
Here this probs makes the intent easier to see
![Screenshot from 2022-04-22 18-42-07](https://user-images.githubusercontent.com/58050969/164817705-5e556239-1899-409b-a9a8-2655eb929854.png)